lib/models/ObjectMD.js

```diff
@@ -1028,6 +1030,33 @@ class ObjectMD {
         return this;
     }
 
+    /**
+     * Create or update the microVersionId field
+     *
+     * This field can be used to force an update in MongoDB (in case
+     * no other metadata field changes) or to manage conflicts during
+     * concurrent updates, using conditions on the microVersionId
+     * field.
+     *
+     * It's a base64-encoded field of 64 bits randomly generated
+     *
+     * @return {ObjectMD} itself
+     */
+    updateMicroVersionId() {
+        this._data.microVersionId = crypto.randomBytes(8).toString('base64');
```

Comment:
I would prefer 'hex' than 'base64' because it is more readable (although a bit longer).

Reply:
not convinced, but ok, I don't mind changing it.

Reply:
done